pkg/connector/handlematrix.go

```diff
@@ -98,6 +98,16 @@ func (tc *TwitterClient) HandleMatrixMessage(ctx context.Context, msg *bridgev2.
 			uploadMediaParams.MediaCategory = "dm_gif"
 		}
 
+		if content.MsgType == event.MsgAudio {
+			converted, err := tc.client.ConvertAudioPayload(ctx, data, content.Info.MimeType)
+			if err != nil {
+				return nil, err
+			} else {
+				data = converted
+				sendDMPayload.AudioOnlyMediaAttachment = true
+				uploadMediaParams.MediaCategory = "dm_video"
+			}
```

Comment:
If the type is already video/mp4, it shouldn't try to convert (local bridges won't have ffmpeg available)